src/term.rs

```diff
@@ -144,7 +144,7 @@ pub enum Term {
     /// type variable. In our example, the last cast to `a` finds `Sealed(2, "a")`, while it
     /// expected `Sealed(1, _)`, hence it raises a positive blame.
     #[serde(skip)]
-    Sealed(SealingKey, RichTerm),
+    Sealed(SealingKey, RichTerm, RichTerm),
```

Comment:
Is there a reason to use a `RichTerm` here instead of using `Label` directly? It incurs some pattern matching that is useless and has unreachable cases, I imagine.

```suggestion
    Sealed(SealingKey, Label, RichTerm),
```

Reply:
Using `Label` directly means we have to change the seal operator `%seal%` from a unary operator to a binary operator. I was trying to avoid modifying the parser. But I will have to use a `Label` now, and modify the parser a bit.

Reply:
For what it worth, it's a pretty straightforward change. A few lines of boiler plate here and there, but that should be a small change, and we did this (changing operator arity, or adding new operators) several times already :slightly_smiling_face: